src/styles/_global-nav.scss

```diff
@@ -1,15 +1,19 @@
 // Values from outer project variables
 ///////////////////////////////////////////////////////////////////////////////
 
-$zindex-navbar-fixed:      1030;
-$green: #04AA51;
+$zindex-navbar-fixed: 1030;
 $blue-darker: #337ab7;
 $white: #FFFFFF;
 $gray: #D8D8D8;
 $black: #161616;
 $transition-duration-medium: 400ms;
 $padding-large-vertical: 11px;
-
+$btn-outer-green: #049B4A;
+$btn-hover-green: #048C43;
+$btn-selected-green: #00701B;
+$btn-outer-blue: #0078CA;
+$btn-hover-blue: #0062B0;
```

Comment:
should this be the color assigned to the external link on hover?

Reply:
Thank you!